metricbeat/module/graphite/server/server.go

```diff
@@ -61,7 +62,11 @@ func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
 // Run method provides the Graphite server with a reporter with which events can be reported.
 func (m *MetricSet) Run(reporter mb.PushReporter) {
 	// Start event watcher
-	m.server.Start()
+	err := m.server.Start()
+	if err != nil {
+		logp.Err("%v", err)
```

Comment:
You might find it useful to report the error to the output (e.g. `reporter.Error(err)`). Then you will the info that one of the graphite listeners failed sent to your output.
```
err = errors.Wrap(err, "failed to start graphite server")
logp.Err("%v, err)
reporter.Error(err)
return
```